ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
```

Comment:
Why this file are changed? You did no introduce any new test item, should have no such new job.

Reply:
need not test the 4.12 nightly build ?

Reply:
okay, if you want to test your PR works well in 4.12, you can add this testing.